core/src/test/java/org/infinispan/marshall/MarshalledValueSingleNodeTest.java

```diff
@@ -20,12 +19,12 @@ protected EmbeddedCacheManager createCacheManager() throws Exception {
       return cm;
    }
 
-   @Test(expectedExceptions = MarshallingException.class)
+   @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
Why do we now get an `IllegalArgumentException`? I must have missed this change.

Reply:
Because protostream throws it when it can't marshal something. Before this PR, the GlobalMarshaller was being used and used to threw MarshallingException

Reply:
Maybe we should wrap this inside `ProtoStreamTranscoder#marshall` with a `MarshallingException` and a message. Something like "Transcoder failed to marshall class %s".